modules/ROOT/pages/_partials/data-sync/server-realtime-updates.adoc

```diff
@@ -2,83 +2,117 @@
 = Configuring your server for real-time updates
 
 A core concept of the GraphQL specification is an operation type called `Subscription`, they provide a mechanism for real time updates.
-To enable Subscriptions on the server, create an appropriate schema similar to the following example.
+To enable Subscriptions on the server, these steps must be followed.
+
+* Configure SubscriptionServer using voyager-subscriptions.
+* Configure a Publish Subscribe Mechanism.
+* Define Subscriptions in the Schema and Implement Subscription Resolvers.
+
+A typical Voyager Server setup without subscriptions is shown below.
+
+[source,js]
+----
+const apolloServer = VoyagerServer({
+  typeDefs,
+  resolvers
+})
+
+const app = express()
+apolloServer.applyMiddleware({ app })
+
+app.listen({ port }, () =>
+  console.log(`🚀 Server ready at http://localhost:${port}${apolloServer.graphqlPath}`)
+)
+----
+
+Follow the procedure to enable subscriptions on the server.
 
 .Procedure
 
-. Create a schema to support subscriptions.
+. Configure SubscriptionServer using `@aerogear/voyager-subscriptions`
 +
-For example, in the following schema a `Task` type is defined, followed by required Mutations and Subscriptions.
-When a new task is created, the resolver for that task publishes the result of this mutation to the `TaskCreated` channel.
+[source,js]
+----
+const { createSubscriptionServer } = require('@aerogear/voyager-subscriptions')
+
+const apolloServer = VoyagerServer({
+  typeDefs,
+  resolvers
+})
+
+const app = express()
+apolloServer.applyMiddleware({ app })
+
+const server = app.listen({ port }, () =>
+  console.log(`🚀 Server ready at http://localhost:${port}${apolloServer.graphqlPath}`)
+
+  createSubscriptionServer({ schema: apolloServer.schema }, {
+    server,
+    path: '/graphql'
+  })
+)
+----
++
+`createSubscriptionServer` installs handlers for managing websocket connections and delivering subscriptions on the server. It is a thin wrapper around `SubscriptionServer` from the popular https://npm.im/subscriptions-transport-ws[subscriptions-transport-ws] module that provides integrations with other modules such as `@aerogear/voyager-keycloak`.
+`createSubscriptionServer` returns a `SubscriptionServer` instance and it supports all of the same arguments and options.
```

Comment:
same options as https://npm.im/subscriptions-transport-ws ?

Reply:
Kind of but not exactly. The same options as the `SubscriptionServer` class from the `subscriptions-transport-ws` module (which has a couple of classes).